doc/sphinx/source/examples/Tutorials/Two_Way_Wave_Equation/Two_Way_Wave_Equation.md

```diff
@@ -0,0 +1,219 @@
+```
+A                                                                 B
+ <-----dx-----> <-----dx----->  ...  <-----dx-----> <-----dx----->
+|----cell 1----|----cell 2----| ... |----cell N-1--|----cell N----|
+0              1              2 ... N-1             N             N+1
+```
+
+For example, from 0 to 1 with 5 cells has six values (x0,x1,x2,x3,x4,x5) each 0.2 away from each other:
+```
+0.0   0.2   0.4   0.6   0.8   1.0
+ o-----o-----o-----o-----o-----o
+ x0    x1    x2    x3    x4    x5
+```
+### Mimetic Difference Grid (Staggered)
+
+Mimetic operators use a staggered grid—half-step offsets at boundaries improve conservation and symmetry. Here, note that **cell 1** and **cell N** are not the sam size as the internal cells, and that there are now **N+2** points.
+```
+A                                                         B
+ <--dx/2--> <-----dx----->  ...  <-----dx-----> <--dx/2-->
+|--cell 1--|----cell 2----| ... |----cell N-1--|--cell N--|
+0          1              2 ... N             N+1        N+2
+```
+
+For the same interval (0–1) with 5 cells, a staggered grid will have smaller dt/s at the beginning and end, and also one mroe point (x0,x1,x2,x3,x4,x5,x6):
+```
+0.0   0.1   0.3   0.5   0.7   0.9   1.0
+ o-----o-----o-----o-----o-----o-----o
+ x0    x1    x2    x3    x4    x5    x6
+```
+
+This staggered layout provides **better discrete analogs** of differential operators, leading to improved conservation and often reduced numerical dispersion.
+
+---
+
+## Convergence and Accuracy
+
+Both FD and MD methods use a **second-order accurate** CTCS scheme in time.  
+The mimetic method, however, allows **higher-order spatial accuracy** by increasing the mimetic operator order `k`.
+
+| Scheme | Spatial Order | Time Order | Conservation | Grid Type |
+|---------|----------------|-------------|---------------|------------|
+| Finite Difference | 2 | 2 | Approximate | Uniform |
+| Mimetic Difference | 2 (or higher) | 2 | Exact (discrete) | Staggered |
+
+---
+
+## Practical Notes
+
+- This setup isolates **spatial discretization effects**—no boundary reflections or external forcing.  
+- Increasing `N` refines the grid and reduces spatial error.  
+- For fair comparison, both solvers should use the same `dt`, `dx`, and runtime length.  
+- The mimetic operator’s flexibility allows easy order-of-accuracy experiments by simply changing `k` in `mimetic_diff_two_way_wave_eq.m`.
+
+---
+
+## Summary
+
+| Feature | Finite Difference | Mimetic Difference |
+|----------|------------------|--------------------|
+| Grid | Uniform | Staggered |
+| Operator | Explicit stencil | Discrete mimetic operator |
+| Conservation | Approximate | Exact (discrete) |
+| Flexibility | Fixed 2nd order | Adjustable order (k) |
+| Accuracy | 2nd order | ≥ 2nd order |
+| Ease of implementation | Simple | Slightly more complex |
+| Best use case | Quick prototyping | Physically consistent PDE solvers |
+
+---
```

Comment:
The description is very nice but we're not using the `literalinclude` clause to actually show snippets of code. 
We should also investigate if an automatic execution and compilation of the tutorial is possible (showing any output, such as plots), like it is for Jupyter notebook for instance or dynamic languages such as python and Julia